src/utils/tests.js

```diff
@@ -4,6 +4,22 @@
 
 import { INVALID_PARAMS_NOT_DEFINED_ALLOWED } from './invalid';
 
+function testEnumIntegrity(enumToTest :Object, expectedEnum :Object) {
+
+  test('should contain only expected enum values', () => {
```

Comment:
Does this kind of programmatic test capture the correct intention? This would pass if I added 

> BANANA: 'BANANA'
 or 
> SUCCESSIZZLE: 'SUCCESSIZZLE'

to the enum wouldn't it?

Reply:
I'm probably misunderstanding, but I think the answer is no:

![Screen Shot 2019-05-10 at 10 44 16 AM](https://user-images.githubusercontent.com/440299/57545990-a798f100-7310-11e9-85ee-baeb86d04b23.png)